tools/eslint-plugin-minecraft-linting/src/Rules/AvoidUnnecessaryCommand.ts

```diff
@@ -305,7 +305,7 @@ const AvoidUnnecessaryCommand = ESLintUtils.RuleCreator(() => 'https://microsoft
         docs: {
             description:
                 'Recommends using a script API if there is a script API (or APIs) that provide 1:1 functionality parity with a command that is used.',
-            recommended: 'error',
+            recommended: 'recommended',
```

Comment:
Is this an intentional change?

Reply:
yeah, the non-experimental utils on update actually have a different type here, and this field is an indication of whether this rule is recommended, strict, or stylistic. I went with recommended.